advanced-achievements-plugin/src/main/resources/config.yml

```diff
@@ -774,6 +774,9 @@ AdvancementsCompleted:
     Message: 20 advancements completed!
     Name: advancements_20
     DisplayName: Game Block Advance
+    # Use semicolons for multiple item rewards as well. The below example will reward 8 diamond and 1 cake
+    Reward:
+      Item: diamond 8;cake 1
```

Comment:
The regex `MULTIPLE_REWARDS_SPLITTER` handles spaces. I think it's more readable if you write it with a space, i.e. `diamond 8; cake 1`.